opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -54,6 +59,48 @@ def collect(self) -> Optional[_PointVarT]:
         pass
 
 
+class _AggregationFactory(ABC):
+    @abstractmethod
+    def _create_aggregation(self, instrument: Instrument) -> _Aggregation:
+        """Creates an aggregation"""
+
+
+class DefaultAggregation(_AggregationFactory):
+    def _create_aggregation(self, instrument: Instrument) -> _Aggregation:
```

Comment:
Should this be public?

Reply:
No, we should minimize our API and keep that private, it is used only internally in the SDK.

Reply:
From an end use perspective I would like to get the default aggregation for instrument, How does public class `DefaultAggregation` without any public methods helpful?

Reply:
We can (and should, I'll add that to this PR) add documentation, I just don't think the user will need to create an aggregation instance from an instance of `DefaultAggregation`.

Reply:
Added documentation